doc/sphinx/grammar.rst

```diff
@@ -0,0 +1,2247 @@
+                 : lra_Q `tactic`
+                 : lra_R `tactic`
+                 : psatz_R `int_or_var` `tactic`
+                 : psatz_R `tactic`
+                 : psatz_Q `int_or_var` `tactic`
+                 : psatz_Q `tactic`
+                 : nsatz_compute `constr`
+                 : omega
+                 : omega with `ident_list`
+                 : omega with *
+                 : protect_fv `string` in `ident`
+                 : protect_fv `string`
+                 : ring_lookup `tactic_expr0` [ `constr_list_opt` ] `constr_list`
+                 : field_lookup `tactic` [ `constr_list_opt` ] `constr_list`
+                 : YouShouldNotTypeThis `ssrintrosarg`
+                 : by `ssrhintarg`
+                 : YouShouldNotTypeThis do `ssrdoarg`
+                 : YouShouldNotTypeThis `ssrtclarg` `ssrseqdir` `ssrseqarg`
+                 : clear `natural`
+                 : move `ssrmovearg` `ssrrpat`
+                 : move `ssrmovearg` `ssrclauses`
+                 : move `ssrrpat`
+                 : move
+                 : case `ssrcasearg` `ssrclauses`
+                 : case
+                 : elim `ssrarg` `ssrclauses`
+                 : elim
+                 : apply `ssrapplyarg`
+                 : apply
+                 : exact `ssrexactarg`
+                 : exact
+                 : exact <: `lconstr`
+                 : congr `ssrcongrarg`
+                 : ssrinstancesofruleL2R `ssrterm`
+                 : ssrinstancesofruleR2L `ssrterm`
+                 : rewrite `ssrrwargs` `ssrclauses`
+                 : unlock `ssrunlockargs` `ssrclauses`
+                 : pose `ssrfixfwd`
+                 : pose `ssrcofixfwd`
+                 : pose `ssrfwdid` `ssrposefwd`
+                 : set `ssrfwdid` `ssrsetfwd` `ssrclauses`
+                 : abstract `ssrdgens`
+                 : have `ssrhavefwdwbinders`
+                 : have suff `ssrhpats_nobs` `ssrhavefwd`
+                 : have suffices `ssrhpats_nobs` `ssrhavefwd`
+                 : suff have `ssrhpats_nobs` `ssrhavefwd`
+                 : suffices have `ssrhpats_nobs` `ssrhavefwd`
+                 : suff `ssrsufffwd`
+                 : suffices `ssrsufffwd`
+                 : wlog `ssrhpats_nobs` `ssrwlogfwd` `ssrhint`
+                 : wlog suff `ssrhpats_nobs` `ssrwlogfwd` `ssrhint`
+                 : wlog suffices `ssrhpats_nobs` `ssrwlogfwd` `ssrhint`
+                 : without loss `ssrhpats_nobs` `ssrwlogfwd` `ssrhint`
+                 : without loss suff `ssrhpats_nobs` `ssrwlogfwd` `ssrhint`
+                 : without loss suffices `ssrhpats_nobs` `ssrwlogfwd` `ssrhint`
+                 : gen have `ssrclear` `ssr_idcomma` `ssrhpats_nobs` `ssrwlogfwd` `ssrhint`
+                 : generally have `ssrclear` `ssr_idcomma` `ssrhpats_nobs` `ssrwlogfwd` `ssrhint`
+                 : ssrinstancesoftpat `cpattern`
+   bindings_list : `bindings_list` , `bindings`
+                 : `bindings`
+   rena
```

Comment:
I think this spelling was deprecated a while ago. It could be time to remove it. In any case, the deprecated annotations would be useful indeed.

Reply:
> I think this spelling was deprecated a while ago.

Which bit are you referring to?

Reply:
Ocaml. It seems that my review was not appropriately attached to the code, probably because you pushed while I was doing it.